viewing_party/party.py

```diff
@@ -1,23 +1,127 @@
 # ------------- WAVE 1 --------------------
 
+from typing import Counter
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if bool(title) == True and bool(genre) == True and bool(rating) == True:
+        return new_movie

```

Comment:
Since this variable is only used once you can just return it directly here:

```suggestion
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }

```